makefile

```diff
@@ -2,6 +2,7 @@ IMAGE_NAME ?= context-aware-config
 DOCKER_DNS ?= localhost
 TENANT ?= dev
 SHELL := /usr/bin/env bash
+FEATURES ?= ssr
```

Comment:
can we mention somewhere that we need to add high-performance mode in the features here if someone wants to run it? 
Maybe we can put a comment as an example .

Reply:
@sauraww I will update the docs, will also have to update docker file to be dynamic. So I will raise another PR for that.